### DISCUSSION TITLE
winget doesnt' find any package

### DISCUSSION THREAD

**Discussion by alcaraz-fran** [microsoft/winget-pkgs#122463] [General] [answered]
I am trying to use winget to install some software like Powershell 7 but the tool doesn't find any package.
For example: if I try the command «winget search Microsoft.PowerShell» the response is «  \».

![image](https://github.com/microsoft/winget-pkgs/assets/73292476/a52acf4d-1b92-4501-8938-86c59c3b0872)

**Comment by mdanish-kh** [comment, marked as answer]
What version of WinGet are you using? You can use `winget -v` to find out. This is likely related to CDN migration that broke some older versions of WinGet.
- https://github.com/microsoft/winget-cli/issues/3652

Please try to upgrade to the latest client either from Microsoft Store, [GitHub Releases](https://github.com/microsoft/winget-cli/releases) or this URL https://aka.ms/getwinget.

**Reply by alcaraz-fran** [reply]
Thank you very much. I have installed the last version and It is fixed.
